tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/OwinIisWebApi2Tests.cs

```diff
@@ -0,0 +1,150 @@
+            : base(iisFixture, output, enableRouteTemplateResourceNames: false)
+        {
+        }
+    }
+
+    [Collection("IisTests")]
+    public class OwinIisWebApi2TestsCallTargetWithFeatureFlag : OwinIisWebApi2Tests
+    {
+        public OwinIisWebApi2TestsCallTargetWithFeatureFlag(IisFixture iisFixture, ITestOutputHelper output)
+            : base(iisFixture, output, enableRouteTemplateResourceNames: true)
+        {
+        }
+    }
+
+    [Collection("IisTests")]
+    public class OwinIisWebApi2TestsCallTargetWithRouteTemplateExpansion : OwinIisWebApi2Tests
+    {
+        public OwinIisWebApi2TestsCallTargetWithRouteTemplateExpansion(IisFixture iisFixture, ITestOutputHelper output)
+            : base(iisFixture, output, enableRouteTemplateResourceNames: true, enableRouteTemplateExpansion: true)
+        {
+        }
+    }
+
+    [UsesVerify]
+    public abstract class OwinIisWebApi2Tests : TracingIntegrationTest, IClassFixture<IisFixture>
+    {
+        private readonly IisFixture _iisFixture;
+        private readonly string _testName;
+
+        public OwinIisWebApi2Tests(IisFixture iisFixture, ITestOutputHelper output, bool enableRouteTemplateResourceNames, bool enableRouteTemplateExpansion = false)
+            : base("Owin.Iis.WebApi2", @"test\test-applications\aspnet", output)
+        {
+            SetServiceVersion("1.0.0");
+            SetEnvironmentVariable(ConfigurationKeys.FeatureFlags.RouteTemplateResourceNamesEnabled, enableRouteTemplateResourceNames.ToString());
+            SetEnvironmentVariable(ConfigurationKeys.ExpandRouteTemplatesEnabled, enableRouteTemplateExpansion.ToString());
+
+            _iisFixture = iisFixture;
+            _iisFixture.ShutdownPath = "/shutdown";
+
+            // OWIN can only run in integrated mode
+            _iisFixture.TryStartIis(this, IisAppType.AspNetIntegrated);
+
+            _iisFixture = iisFixture;
+            _testName = nameof(AspNetWebApi2Tests) // Note that these spans are identical to the non-owin webapi2 version
+                      + ".Integrated"
+                      + (enableRouteTemplateExpansion ? ".WithExpansion" :
+                        (enableRouteTemplateResourceNames ?  ".WithFF" : ".NoFF"));
+        }
+
+        public static TheoryData<string, int, int> Data() => new()
+        {
+            { "/api/environment", 200, 2 },
+            { "/api/absolute-route", 200, 2 },
+            { "/api/delay/0", 200, 2 },
+            { "/api/delay-optional", 200, 2 },
+            { "/api/delay-optional/1", 200, 2 },
+            { "/api/delay-async/0", 200, 2 },
+            { "/api/transient-failure/true", 200, 2 },
+            { "/api/transient-failure/false", 500, 3 },
+            { "/api/statuscode/201", 201, 2 },
+            { "/api/statuscode/503", 503, 2 },
+            { "/api/constraints", 200, 2 },
+            { "/api/constraints/201", 201, 2 },
+            // { "/api/TransferRequest/401", 401, 4 },
```

Comment:
nit: Could you add a comment here to explain why it was here/why we're ignoring it for these tests?